### PR TITLE
for chaste code generation I would like to stay closer to order in th…

### PR DESCRIPTION
**Pull request by MauriceHendrix**
for chaste code generation I would like to stay closer to order in the model to stay closer to pycml output, therfore I needed a version of get_equations_for that does not do a lexicographical_sort

### CODE REVIEW

File: cellmlmanip/model.py [hunk 1]
```diff
@@ -410,10 +410,14 @@ def get_equation_graph(self, refresh=False) -> nx.DiGraph:
         self.graph = graph
         return graph
 
-    def get_equations_for(self, symbols):
-        """Get all equations for given collection of symbols"""
+    def get_equations_for(self, symbols, lexicographical_sort=True):
+        """Get all equations for given collection of symbols
```
Comment (MichaelClerx) [inline cellmlmanip/model.py]: Docstrings should have the format

    """
    One-line description

    Longer description, which may include a list of arguments (in a syntax that's unfortunately project-specific).
    The language for docstrings is "reStructuredText", in which you indicate variable names (and other code) with two backticks, like ``this``
    """

**Comment by MichaelClerx** [comment]
What order would you like them in instead then? Make sure it's deterministic, so you always get the same output in tests

**Comment by jonc125** [comment]
Pycml certainly ensures deterministic output. I can't recall what ordering it applies to equations beyond a topographical sort though.

**Comment by MauriceHendrix** [comment]
> Needs better docstring and more unit tests --> right now cover is reduced

that's why this is a draft I'm working on that

**Pull request closed without merging**